docs/environments/mainnet/config.json

```
@@ -5,6 +5,8 @@
   "ApplicationVersion": 1,
   "ByronGenesisFile": "byron-genesis.json",
   "ByronGenesisHash": "5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb",
+  "ConwayGenesisFile": "conway-genesis.json",
+  "ConwayGenesisHash": "f28f1c1280ea0d32f8cd3143e268650d6c1a8e221522ce4a7d20d62fc09783e1",
```

This would be the hash.
```suggestion
  "ConwayGenesisHash": "011788fdf180fd6f0fe2d9916b8345c289ffa093014805cbc681344be7200926",
```